tools/driver/p4c/config.py

```diff
@@ -0,0 +1,115 @@
+
+from __future__ import absolute_import
+import sys
+import os
+
+class Config(object):
+    """
+    Config - Configuration data for a 'p4c' tool chain.
+
+    The common configuration data include compiler/assembler name,
+    compiler/assembler installation path.
+    """
+
+    def __init__(self, config_prefix):
+        self.config_prefix = config_prefix or 'p4c'
+        self.site_config_name = '%s.site.cfg' % (self.config_prefix)
```

Comment:
You never refer to this.

In general, there are a lot of comments that suggest that `site.cfg` is special, but it seems like the code now just searches for `.cfg` files. You should update the comments and clarify what's happening.